tests/RetryTest.php

```diff
@@ -196,7 +196,6 @@ private function setupLargeTable(): void
     {
         $tableName = 'sales';
         $temp = new Temp();
-        $temp->initRunFolder();
```

Comment:
Je otázka, jestli by to tady nemělo na konci smazat. Ale asi je to v pohodě - prostě se to smaže systémem "někdy" z /tmp

Reply:
je to jen v testu, tak by to nemělo ničemu vadit

Reply:
pro úplnost jsem to tam ještě přidal